test/CalculatorGTest.cpp

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file CalculatorGTest.cpp
+* @brief unit tests for the calculator
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include "../inc/Calculator.h"
```

Comment:
Figure out a way using CMake so you don't need a relative path here... doing `#include "Calculator.h"` is also valid... (It might already be like this)